axolotl-web/src/layouts/Default.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="base-layout">
+    <header :class="currentRoute + ' header fixed-top'">
+      <div class="container">
+        <div class="header-row row">
+          <div class="col-2 header-left">
+            <button
+              v-if="$route.meta.hasBackButton"
+              class="btn"
+              @click="back()"
+            >
+              <font-awesome-icon icon="arrow-left" />
+            </button>
+          </div>
+          <div class="col-8 header-center">
+            <slot name="header" />
+          </div>
+          <div class="col-2 header-right justify-content-end d-flex">
+            <div class="dropdown">
+              <button
+                id="dropdownMenuButton"
+                class="btn btn-primary"
+                type="button"
+                data-bs-toggle="dropdown"
+                aria-expanded="false"
+              >
+                <font-awesome-icon icon="ellipsis-v" />
+              </button>
+              <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                <slot name="menu" />
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main class="container">
+      <slot />
+    </main>
+    <footer>
+      <slot name="footer" />
+    </footer>
+  </div>
+</template>
+<script>
+export default {
+  name: "DefaultLayout",
+  data: () => ({
+    showMenu: false,
+  }),
+  computed: {
+    currentRoute() {
+      return this.$route.name;
+    },
+  },
+  methods: {
+    back() {
+      this.$router.go(-1);
+    },
+    toggleMenu() {
+      this.showMenu = !this.showMenu;
+    },
+  },
+};
+</script>
+<style scoped>
+.header {
+  padding: 5px 0;
+  background-color: #2090ea;
+  z-index: 2;
+  -webkit-box-shadow: 0px -11px 14px 7px rgba(0, 0, 0, 0.75);
```

Comment:
I understand that these days the browser prefix (-webkit-, -moz-) is no longer needed for box-shadow.

Here two references:

* https://caniuse.com/?search=box-shadow
* https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

Reply:
thanks for the hint!